manifests/service.pp

```diff
@@ -26,8 +26,8 @@
       }
       ~> Service['caddy']
     }
-    'redhat': { # we could probably add 'debian' for older debian releases but not sure
-      file {'/etc/init.d/caddy':
+    'redhat': {
+      file { '/etc/init.d/caddy':
         ensure  => file,
         mode    => '0744',
```

Comment:
Unrelated, but what a weird mode. `0755` is much more logical IMHO.

Reply:
Yeah, this was already in here and nobody cared about it. I will change it. ;)